CONTRIBUTING.md

```diff
@@ -208,6 +208,44 @@ cargo test
 
 from the `rustworkx-core` directory.
 
+### Fuzz Testing in rustworkx
+
+We use cargo-fuzz to test rustworkx for unexpected crashes or undefined behavior. Follow these steps to run fuzzing locally.
+
+#### Building Fuzz Targets
+To build the fuzzing targets, first install cargo-fuzz:
+```sh
+cargo install cargo-fuzz
+```
+then run the following command:
+
+```sh
+cargo fuzz build
+```
+#### To run a fuzz test (e.g., is_connected_fuzz):
+
+```sh
+cargo fuzz run is_connected_fuzz
+```
+For nightly toolchain:
+```sh
+cargo +nightly fuzz run is_connected_fuzz
```

Comment:
Is there a way to run all the fuzz targets? If not, we should point users to Cargo.toml with the list of targets.